packages/nuxt/src/core/features.ts

```diff
@@ -39,14 +36,30 @@ export async function ensurePackageInstalled (
 
   logger.info(`Installing ${name}...`)
   try {
-    await addDependency(name, {
-      cwd: options.rootDir,
-      dev: true
-    })
+    await installCommand()
     logger.success(`Installed ${name}`)
     return true
   } catch (err) {
     logger.error(err)
     return false
   }
 }
+
+// TODO: refactor to Nuxi
+const installPrompts = new Set<string>()
+export function installNuxtModule (name: string, options?: EnsurePackageInstalledOptions) {
+  if (installPrompts.has(name)) { return }
+  installPrompts.add(name)
+  const nuxt = useNuxt()
+  return promptToInstall(name, async () => {
+    const { runCommand } = await import('nuxi')
+    await runCommand('module', ['add', name, '--cwd', nuxt.options.rootDir])
+  }, { rootDir: nuxt.options.rootDir, searchPaths: nuxt.options.modulesDir, ...options })
```

Comment:
We could ensure `import.meta.url` is in the search path. It refers to the `nuxt` package that has direct dependency on `nuxi`. In this case, it would be almost impossible that it won't work or be found...

Also, since rootDir change is happening, was `--cwd` flag also necessary.

Reply:
Yes, it was necessary; it was otherwise picking up the parent directory in a monorepo.

Reply:
what makes you think that this is a serious bug? we're not executing this nuxi command within the rootDir...

Reply:
Not for nuxt but for nuxi which i like to investigate. If `nuxi` CLI is executed with (OS) CWD of `rootDir`, it should run `module add` also within it not wrongly to the monorepo hoisted value. (or perhaps the `runComand` itself could accept CWD for programmatic API) 

Also i guess now would make sense we expose `runCommand` from kit wdyt?

Both above and this are unrelated to PR but worth to track)

Reply:
Could we support passing arbitrary args as part of options, perhaps?